pom.xml

```diff
@@ -126,6 +128,11 @@
                 <artifactId>fest-assert</artifactId>
                 <version>${version.fest-assert}</version>
             </dependency>
+            <dependency>
+                <groupId>org.assertj</groupId>
+                <artifactId>assertj-core</artifactId>
```

Comment:
I prefer to change it in another PR, so it is more trackable. The PR is about adding Gradle support not to refactor tests of whole project.